lblod/frontend-gelinkt-notuleren
# Switch to location-plugin with map for address variables

### Overview
Uses the new nodespec and components from the location plugin for addresses. Any address variables from templates get correctly picked up by the plugin and treated as locations instead.

Includes an update to ember-power-select as the old version caused a mismatch in a mutual dependency.

##### connected issues and PRs:
https://binnenland.atlassian.net/browse/GN-4840

### Setup
N/A

### How to test/reproduce
Create/edit an agenda point or regulatory statement, insert locations, edit them, jump between 'places' and 'addresses', etc. Create a regulatory statement from a template including an address variable to confirm that it's still functional as a location.
Keep an eye out for any problems with power-select driven components.

### Challenges/uncertainties
N/A

### Checks PR readiness
